README.md

```diff
@@ -103,50 +157,6 @@ Run all the test cases for all the components within the repo.
 
 Run all the test cases for all the components within the repo and provides a coverage report.
 
-## Resources
```

Comment:
Why did the list of resources get removed?

Reply:
I removed this since they are mentioned in the CloudFormation template files which the user will eventually run. Should I add it back?

Reply:
Just making sure that it was an intentional removal.